org.bleachbit.BleachBit.json

```
@@ -0,0 +1,63 @@
+{
+  "app-id": "org.bleachbit.BleachBit",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
```

Update runtime to 20.08
```suggestion
  "runtime-version": "20.08",
```